Phonebook.Frontend/src/app/modules/profile-picture/components/profile-picture/profile-picture.component.ts

```diff
@@ -65,7 +65,22 @@ export class ProfilePictureComponent implements OnInit, OnDestroy {
         }
       });
   }
-
+  private getAprilProfilePictureUrl() {
+    var random = Math.floor(Math.random() * Math.floor(9)).toString();
+    switch (random) {
+      case '0': return 'https://cdn.pixabay.com/photo/2020/02/15/17/51/drawing-4851591_960_720.png';
```

Comment:
```suggestion
      case 0: return 'https://cdn.pixabay.com/photo/2020/02/15/17/51/drawing-4851591_960_720.png';
```
Man kann die übrigens auch mathematisch vergleichen